src/core/StatedBeanContainer.ts

```diff
@@ -123,6 +131,7 @@ export class StatedBeanContainer extends Event {
     });
   }
 
+  // @internal
   async observeBeanField(
```

Comment:
If it is considered as `internal`, `private async _observeBeanField` would be bettwe.